xml/System/Environment.xml

```diff
@@ -1753,13 +1753,20 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of processors on the current machine.</summary>
-        <value>The 32-bit signed integer that specifies the number of processors on the current machine. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
+        <summary>Gets the number of processors available to the current process.</summary>
+        <value>The 32-bit signed integer specifies the number of processors that are available.</value>
```

Comment:
> The wording reads well to me.

Please look at the other properties of the `Environment` class or any other well-known Framework class for that matter. There is an implicit "is" between the "Property Value" and its description.  "Value is the 32-bit signed integer specifiers..." is not grammatically correct.

Reply:
```suggestion
        <value>The 32-bit signed integer that specifies the number of processors that are available.</value>
```